### PR TITLE
feat(TargetRendering/Nextgen): New mode: Ghost

### PR DESCRIPTION
**Pull request by sqlerrorthing**
https://github.com/user-attachments/assets/a752a1d6-f040-4ace-b8f1-417af0c1625f

### CODE REVIEW

**Comment by superblaubeere27** [comment]
looks cool

**Comment by liquidsquid1** [comment]
this looks super cool! I wonder if other cool things can be done utilising the image glow.

**Comment by ghost** [comment]
Pull this RN

File: src/main/kotlin/net/ccbluex/liquidbounce/utils/render/TargetRenderer.kt [hunk 1]
```diff
@@ -68,15 +78,147 @@ class WorldTargetRenderer(module: ClientModule) : TargetRenderer<WorldRenderEnvi
...
     val glowingCircle = GlowingCircle(module)
+    val ghost = Ghost()
 
     override val appearance =
         choices<Choice>(
             module,
             "Mode",
-            { glowingCircle },
-            { arrayOf(legacy, circle, glowingCircle) }
+            { ghost },
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/utils/render/TargetRenderer.kt]: Default should stay on glowing circle, thanks.

Comment (sqlerrorthing) [reply src/main/kotlin/net/ccbluex/liquidbounce/utils/render/TargetRenderer.kt]: okay

**Pull request merged**